system/Config/AutoloadConfig.php

```diff
@@ -143,6 +143,7 @@ public function __construct()
 			'CodeIgniter\Debug\Exceptions'           => BASEPATH.'Debug/Exceptions.php',
 			'CodeIgniter\Debug\Timer'                => BASEPATH.'Debug/Timer.php',
 			'CodeIgniter\Debug\Iterator'             => BASEPATH.'Debug/Iterator.php',
+			'CodeIgniter\Encryption\Encryption'               => BASEPATH.'Encryption/Encryption.php',
```

Comment:
Alignment is off, but that's not a big deal, since I'm becoming more curious if pre-loading these paths actually helps speed. Just haven't had a chance to do a real test just yet.